mottak/src/main/kotlin/no/nav/melosys/soknadmottak/mottak/MottakService.kt

```diff
@@ -53,6 +55,7 @@ class MottakService(
                         innsendtTidspunkt
                     )
                     if (soknadService.erSøknadArkivIkkeLagret(arkivRef)) {
+                        Metrics.counter(SØKNAD_MOTTATT).increment()
```

Comment:
Flytte den et par hakk ned? Til etter `fjernElementFraKø` feks?

Reply:
Det kommer an på hva vi teller, kan man si.

Reply:
Jeg tenkte vi burde ha en teller så tidlig som mulig i mottak for å kunne fange opp at vi f.eks. har mottatt flere søknader enn vi har kvittert for?

Reply:
Ja ser den. Men hva med etter vi faktisk har lagret søknaden? For da har vi jo faktisk mottatt den, og vi vil ikke telle samme søknad på nytt igjen pga `soknadService.erSøknadArkivIkkeLagret(arkivRef)`?

Reply:
Ja, det kan jeg godt gjøre. Mange av problemene vi har opplevd nå i innspurten har inntruffet når vi forsøker å lagre pdf, men nå er det flytta til etter lagring, så da bør det være uproblematisk å lagre først.